org.eclipse.mylyn-site/pom.xml

```diff
@@ -27,7 +27,7 @@
   <name>Mylyn for Eclipse 4.x</name>
 
   <properties>
-    <eclipse.repo>https://download.eclipse.org/releases/2023-06</eclipse.repo>
+    <eclipse.repo>https://download.eclipse.org/releases/2024-09</eclipse.repo>
```

Comment:
Sorry about this.  Other places I used /latest rather than a fixed version.

Reply:
Nothing to sorry about, things are moving on. I'll change it to `https://download.eclipse.org/releases/latest`